sale_order_import_ubl/wizard/sale_order_import.py

```diff
@@ -0,0 +1,137 @@
+# © 2016-2017 Akretion (Alexis de Lattre <alexis.delattre@akretion.com>)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import api, models
+from odoo.tools import float_is_zero
+
+
+class SaleOrderImport(models.TransientModel):
+    _name = "sale.order.import"
+    _inherit = ["sale.order.import", "base.ubl"]
+
+    @api.model
+    def parse_xml_order(self, xml_root, detect_doc_type=False):
+        start_tag = "{urn:oasis:names:specification:ubl:schema:xsd:"
+        rfq = "RequestForQuotation"
+        if xml_root.tag == start_tag + "Order-2}Order":
+            if detect_doc_type:
+                return "order"
+            else:
+                return self.parse_ubl_sale_order(xml_root)
+        elif xml_root.tag == "{}{}-2}}{}".format(start_tag, rfq, rfq):
```

Comment:
what does this "2" for?

Reply:
I don't know, it respects the UBL specs I suppose, I didn't change that.